keymanager/Cargo.toml

```diff
@@ -15,7 +15,8 @@ fs-err = { workspace = true }
 futures = { workspace = true }
 helper_functions = { workspace = true }
 itertools = { workspace = true }
-log = { workspace = true }
+logging = { workspace = true }
+tracing = { workspace = true }
```

Comment:
[**chore**](https://conventionalcomments.org/): Keep dependencies in alphabetical order